Libraries/TranscriptTestRunner/TestRunner.cs

```diff
@@ -27,6 +28,7 @@ public class TestRunner
         private readonly ILogger _logger;
         private readonly int _replyTimeout;
         private readonly TestClientBase _testClient;
+        private Dictionary<string, string> _scriptParams;
```

Comment:
I don't think we need a field to store this, we can just take it in RunTestAsync and pass it as a parameter to ExecuteTestScriptAsync (it doesn't seem to be used again)